apps/worker/src/app/workflow/usecases/send-message/send-message-chat.usecase.ts

```diff
@@ -108,6 +110,14 @@ export class SendMessageChat extends SendMessageBase {
         Object.values(ChatProviderIdEnum).includes(chan.providerId as ChatProviderIdEnum)
       ) || [];
 
+    const phone = subscriber.phone;
+    chatChannels.push({
+      providerId: ChatProviderIdEnum.WhatsAppBusiness,
+      credentials: {
+        phoneNumber: phone,
+      },
+    });
```

Comment:
I don't love this implementation, but until we refactor the whole "Chat" channel flow, this workaround is needed.